Library/Formula/libhttpserver.rb

```diff
@@ -0,0 +1,55 @@
+class Libhttpserver < Formula
+  homepage "https://github.com/etr/libhttpserver"
+  url "https://github.com/etr/libhttpserver/archive/v0.7.1.tar.gz"
+  head "https://github.com/etr/libhttpserver.git"
+  sha1 "b36c8a5f523c04613d6a36c86bbb31a2b92aab83"
+
+  option "with-cpp11", "Enable c++11 std classes"
+  depends_on "libmicrohttpd"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  option :universal
+  patch :p0, :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of what it does in a comment in the formula file. Thanks!